omero/sysadmins/unix/server-install-homebrew.rst

```diff
@@ -41,155 +51,196 @@ running::
+In order to develop on OMERO, we recommend you ensure you have your Mac setup for
+development. The first step to achieving this is to create a :file:`.bash_profile` file in the
+root directory of your user folder.
 
-Install Homebrew and make sure :file:`/usr/local/bin` is prepended to your
-:envvar:`PATH`::
+To create a :file:`.bash_profile` from terminal, if one does not already exist::
 
-    $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-    $ export PATH=/usr/local/bin:$PATH
+    $ touch ~/.bash_profile
 
-Update Homebrew and run 'brew doctor' to fix potential issues beforehand::
+To open your :file:`.bash_profile` in a text editor, such as the built-in TextEdit app, use::
 
-    $ brew update
-    $ brew doctor
+    $ open -a TextEdit.app ~/.bash_profile
 
-Install git if not already present::
+.. note::
+   If you want to see changes to your :file:`.bash_profile` take effect without restarting
+   OS X, run::
 
-    $ brew list | grep "\bgit\b" || brew install git
+   $ source ~/.bash_profile
 
-Install PostgreSQL database server::
+Requirements
+------------
+
+1. Open a command-line terminal and install git if not already present::
+
+    $ brew install git
+
+2. Install PostgreSQL database server::
 
-    $ export LANG=${LANG:-en_US.UTF-8}
-    $ export LANGUAGE=${LANGUAGE:-en_US:en}
     $ brew install postgresql
 
-.. _`Homebrew and Python`: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Homebrew-and-Python.md
+   To ensure PostgreSQL uses UTF-8 encoding, open your bash profile and 
+   add the following enviornment variables::
+
+    export LANG=en_US.UTF-8
+    export LANGUAGE=en_US:en
+
+3. OMERO depends on Ice 3.6 and unfortunately does not run with 
+   the latest version of Ice at this time (Ice 3.7.3). To obtain 
+   Ice 3.6, we need to add a *tap* to Homebrew::
 
-You should install OMERO using Python 2.7 provided by
-Homebrew since it makes using Homebrew-provided modules
-simpler, for example the Ice python bindings needed by OMERO. For a
-more thorough description of the Homebrew solution, see the `Homebrew
-and Python`_ page. Note that the automated script linked above tests
-the OMERO installation using the Homebrew Python.
+    $ brew tap zeroc-ice/tap
+    $ brew install zeroc-ice/tap/ice36
 
-To install the Python provided by Homebrew::
+.. Note::
+   If you already have a version of Ice that is not 3.6 installed, you can instruct Homebrew to *unlink* it using 
+   ```$ brew unlink ice```. You can then instruct Homebrew to link to Ice 3.6 using ```$ brew link ice@36```
+
+4. Install Python provided by Homebrew::
 
     $ brew install python
 
-Check that Python is working and is version 2.7::
+   Homebrew installs Python in the following location::
+
+    '/usr/local/opt/python/libexec/bin'
+
+   Follow the instructions from the brew Python install and set the Homebrew version of Python 
```

Comment:
Need to reformulate this. I guess you actually do not want me to follow the instructions and add the line ``export PATH="/usr/local/opt/python/libexec/bin:$PATH"`` to my .bash_profile. Instead, you suggest I do what is written in the "Note" below and do https://github.com/openmicroscopy/ome-documentation/pull/1766/files#diff-38c0f35540194fd0f8261d5f85fae3fcR121 and https://github.com/openmicroscopy/ome-documentation/pull/1766/files#diff-38c0f35540194fd0f8261d5f85fae3fcR125. You should make it clear that this is the mainline of the installation, if this is so, not let me do both :)